src/Core/Tools/Models/Business/ReferenceEvent.cs

```diff
@@ -243,4 +243,12 @@ public ReferenceEvent(ReferenceEventType type, IReferenceable source, ICurrentCo
     /// the value should be <see langword="null" />.
     /// </value>
     public string SignupInitiationPath { get; set; }
+
+    /// <summary>
+    /// The will hold the details around the plan cusomer is and the plan they are migrating to. For example, "Teams Starter → Teams, Enterprise".
```

Comment:
🎨 Just a suggestion to improve the clarity of these docs.

```suggestion
    /// The upgrade applied to an account. The current plan is listed first,
    /// followed by the plan they are migrating to. For example, 
    /// "Teams Starter → Teams, Enterprise".
```

Reply:
added recommended comment